lib/main.dart

```diff
@@ -11,11 +12,14 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      routes: {
+        '/': (context) => const IndexPage(),
+        '/details': (context) => const MovieDetailsPage(),
```

Comment:
Create a routes class in core to hold all of these routes instead of having to type the strings everywhere you want to do Navigation